cluster/config-defaults.yaml

```diff
@@ -317,7 +320,7 @@ teapot_admission_controller_daemonset_reserved_memory: "64Gi"
 {{if eq .Cluster.Environment "production"}}
 teapot_admission_controller_validate_application_label: "true"
 teapot_admission_controller_validate_base_images: "true"
-teapot_admission_controller_validate_pod_images: "true"
+teapot_admission_controller_validate_pod_images: "false"
```

Comment:
This is not an error, right?

Reply:
No, it's fine. It's a config item that's currently not used and just put in place for later.

Reply:
thanks